Description: Remove a configuration variable that is not used anywhere
 (as of version 0.7.10.1).
Author: Sven Joachim <svenjoac@gmx.de>
Last-Update: 2011-11-08

---
 backup-manager.conf.tpl |   10 ----------
 1 file changed, 10 deletions(-)

--- a/backup-manager.conf.tpl
+++ b/backup-manager.conf.tpl
@@ -42,11 +42,6 @@
 # Number of days we have to keep an archive (Time To Live)
 export BM_ARCHIVE_TTL="5"
 
-# At which frequency will you build your archives?
-# You can choose either "daily" or "hourly". 
-# This should match your CRON configuration.
-export BM_ARCHIVE_FREQUENCY="daily"
-
 # Do you want to purge only the top-level directory or all
 # directories under BM_REPOSITORY_ROOT?
 export BM_REPOSITORY_RECURSIVEPURGE="false"
